content/general/concepts/modulo/modulo.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'Modulo'
+Description: 'A modulo operation takes two arguments and returns the remainder of the first argument divided by the second.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Arithmetic'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+Modulo is a mathematical operation available in every programming language along with more common ones like addition and multiplication. A modulo operation takes two arguments and returns the remainder of the first argument divided by the second. The result of the operation is called the modulus.
+
+Most programming languages use the symbol `%` to represent the modulo operation, though some may use `mod` or other variations.
+
+## Examples
+
+The following is an example of the modulo operator in Python:
+
+```py
```

Comment:
```suggestion
```codebyte/py
```

Reply:
I'm thinking we can turn this into a code byte so we wouldn't need the commented outputs

Reply:
I would also include one breaking example, if possible. Like, what would we get if we tried to mod a negative integer?

```
7 % -2
```